sdk/ledger-service/http-json/src/main/scala/com/digitalasset/http/PackageService.scala

```diff
@@ -352,7 +347,7 @@ object PackageService {
   }
 
   type ResolveTemplateRecordType =
-    ContractTypeId.Template.RequiredPkg => Error \/ typesig.Type
+    ContractTypeId.Template.RequiredPkgId => Error \/ typesig.Type
```

Comment:
After substituting these type aliases, we are changing the argument used to lookup a template record type from
`ContractTypeId.Template[Ref.PackageRef]`
to
`ContractTypeId.Template.[Ref.PackageId]`.

This precludes the possibility of specifying the package with a `Ref.PackageName`, which would fail to find anything.